engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/ordering/SortArrayNode.java

```diff
@@ -0,0 +1,29 @@
+package org.enso.interpreter.node.expression.builtin.ordering;
+
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.AcceptsError;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.state.State;
+
+/** Just a wrapper for {@code Array.sort_builtin} that delegates to {@code Vector.sort_builtin}. */
+@BuiltinMethod(type = "Array", name = "sort_builtin")
```

Comment:
After #6218 there remained only one difference between `Vector` and `Array` - `Array` has additional `sort_buitin`. `Vector` sorting used to be done by copying the `Vector` content into `Array`, sorting that one in place and wrapping it into `Vector`.

`Array.sort_buitin` was (supposed to be) a private method. Implementation of `Array.sort` delegated to `Vector.sort` and returned new copy of the original array.

How that is planned to be done now? Can we have just a single `sort_buitin`? Do we need both for some reason?

Reply:
The only reason why `Array.sort_builtin` exist now is because #6282. When #6282 is fixed, it can be removed. I will put a link to this comment in that issue, so that we know that we should remove this `SortArrayNode` altogether once that issue is resolved.